syntax = "proto3";

service TimeService {
  rpc GetAllShowTimes (Empty) returns (stream ShowTime);
  rpc GetShowTimesByDate (Date) returns (ShowTime);
}


message Empty {}

message Date {
  string date=1;
}

message ShowTime {
  Date date  = 1;
  repeated string movies= 2;
}
